shell/pages/c/_cluster/uiplugins/DeveloperInstallDialog.vue

```diff
@@ -18,17 +18,18 @@ export default {
 
   data() {
     return {
-      name:              '',
-      location:          '',
-      persist:           false,
-      canModifyName:     true,
-      canModifyLocation: true,
-      showModal:         false,
+      name:                '',
+      location:            '',
+      persist:             false,
+      canModifyName:       true,
+      canModifyLocation:   true,
+      showModal:           false,
+      returnFocusSelector: '[data-testid="extensions-page-menu"]'
```

Comment:
This is fine for now, but I still think there's a design problem that we need to solve if we plan on using this more in the future. `returnFocusSelector` requires that components like `DeveloperInstallDialog` have knowledge of parent or sibling components. IMO it would be best if we could have some localized data store that will allow us to specify the `returnFocusSelector` closer the the action that will be triggering the modal.

Reply:
I've created https://github.com/rancher/dashboard/issues/13367 to follow-up with this topic. 

It also covers the part above about the return target for the focus.